Keymaps/Default (Linux).sublime-keymap

```diff
@@ -28,6 +28,15 @@
      // Find Symbol References
     { "keys": ["shift+f12"], "command": "lsp_symbol_references", "context": [{"key": "setting.lsp_active"}]},
 
+    // Next/Prevous diagnostics
+    { "keys": ["f8"], "command": "lsp_next_diagnostic"},
+    { "keys": ["shift+f8"], "command": "lsp_previous_diagnostic"},
+    { "keys": ["escape"], "command": "lsp_hide_diagnostic", "context":
+        [
+            { "key": "setting.lsp_diagnostic_phantom"}
+        ]
+    },
```

Comment:
All these keybindings should have the `{"key": "setting.lsp_active"}` context, otherwise some people could complain  about how this plugin hijacks other plugins' keybindings.